restructured concepts folders (#607)
kyma-project/community: concepts/README.md

# Concepts

> A folder for collecting and working on PoCs and proposed concepts. Having a concept located here does not mean that it gets realized at all or exactly in the described way. 

Reviewer: ```suggestion
This folder contains PoCs and proposed concepts. Having a concept located here does not mean that it gets realized at all or exactly in the described way. 
```
